apps/judicial-system/web/src/routes/Prosecutor/components/PoliceCaseFiles/PoliceCaseFiles.tsx

```diff
@@ -151,7 +152,7 @@ const PoliceCaseFiles: React.FC<Props> = ({
         files: policeData ? policeData.policeCaseFiles : [],
         isLoading: false,
         hasError: true,
-        errorCode: policeDataError?.graphQLErrors[0].extensions?.code,
+        errorCode: policeDataError?.graphQLErrors[0].extensions.code as string,
```

Comment:
```suggestion
        errorCode: policeDataError?.graphQLErrors[0].extensions?.code as string,
```

Extensions is of type `readonly extensions: { [key: string]: any } | undefined;`. It can be `undefined` so I'd think we need a `?` after `extensions`